regex-assembly/933211.ra

```diff
@@ -0,0 +1,30 @@
+##! Please refer to the documentation at
```

Comment:
The contents of this file are identical to the first, except for the suffix, correct? Then put everything, except for the suffix, into an include file and then in both of these files use `##!> include <your include file>`.

Reply:
awesome! tnx